.gitignore

```diff
@@ -147,6 +147,11 @@ examples/hiop/ex9-mesh.*
 examples/hiop/ex9-init.*
 examples/hiop/ex9-final.*
 
+
+examples/ex71
```

Comment:
It will be renamed before merging - I am trying to avoid name changes during the development. I am periodically merging the master branch in the current one.